Gemfile.jruby-3.1.lock.release

```diff
@@ -880,6 +880,7 @@ GEM
 PLATFORMS
   java
   universal-java-11
+  universal-java-17
```

Comment:
@robbavey spidey-senses:

 - you introduced `LogStash::Bundler::genericise_platform` a while ago, and hooked into it with the plugin manager's install/update/remove commands. I remember that the effect was to ensure `java` platform was here and that the other specific ones were not present, but I don't recall _why_ we needed to do it.
 - the current lockfile already has `universal-java-11` (should it?), and this PR adds `universal-java-11` (does this have net-negative effect?).

Reply:
I don't think we want to _add_ to it, and I'm surprised that `universal-java-11` is there too. Given that it has been here since `8.14`, it likely isn't doing any harm, but I don't see a good reason to add `universal-java-17`. 

I don't recall the why - I will investigate when I have a moment today